pkg/thing/interactors/update_data_test.go

```diff
@@ -0,0 +1,144 @@
+package interactors
```

Comment:
There are a couple formatting mistakes in the file, I suggest using `go fmt pkg/thing/interactors/update_data_test.go` :stuck_out_tongue:

Reply:
@lcbm You're right! Done